ketcher-autotests/tests/Structure-Creating-&-Editing/Simple-Objects/action-on-simple-objects.spec.ts

```diff
@@ -0,0 +1,242 @@
+import { Page, test, expect } from '@playwright/test';
+import {
+  LeftPanelButton,
+  openFileAndAddToCanvas,
+  waitForPageInit,
+  waitForRender,
+  drawBenzeneRing,
+  resetCurrentTool,
+  selectLeftPanelButton,
+  takeEditorScreenshot,
+  clickInTheMiddleOfTheScreen,
+  dragMouseTo,
+  getCoordinatesOfTheMiddleOfTheScreen,
+  selectDropdownTool,
+  copyAndPaste,
+  selectTopPanelButton,
+  TopPanelButton,
+  saveToFile,
+  receiveFileComparisonData,
+  pressButton,
+  STRUCTURE_LIBRARY_BUTTON_NAME,
+} from '@utils';
+import { getKet } from '@utils/formats';
+
+const ellipseWidth = 120;
+const ellipseHeight = 100;
+
+const setupEllipse = async (page: Page) => {
+  await selectLeftPanelButton(LeftPanelButton.ShapeEllipse, page);
+  const { x, y } = await getCoordinatesOfTheMiddleOfTheScreen(page);
+  const ellipseCoordinates = { x: x + ellipseWidth, y: y + ellipseHeight };
+  await clickInTheMiddleOfTheScreen(page);
+  await dragMouseTo(ellipseCoordinates.x, ellipseCoordinates.y, page);
+  return ellipseCoordinates;
+};
+
+async function setZoomInputValue(page: Page, value: string) {
+  await page.getByTestId('zoom-input').click();
+  await page.getByTestId('top-toolbar').getByRole('textbox').fill(value);
+  // await page.getByTestId('zoom-value').fill(value);
```

Comment:
this commented line can be removed

Reply:
I know, leave it for now because this is the function which Nikita suggested for Zoom actions but the locator 'zoom-value' is not recognisable in the test even if the locator apparently exists

Reply:
so your plan is to do some updates/refactoring on that?

Reply:
Probably where more locators will be available to use

Reply:
I removed the commented line